app.json

```diff
@@ -33,7 +41,10 @@
   },
   "ios": {
     "bundleIdentifier": "com.shapeShift.shapeShift",
-    "supportsTablet": true
+    "supportsTablet": true,
+    "infoPlist": {
+      "ITSAppUsesNonExemptEncryption": false
```

Comment:
While at it, Apple is asking everytime we upload a build if we use a non standard encryption algorithm, which I guess isn't the case so far, setting this to false is automatically responding "no" to this